templates/library-package/package.json

```diff
@@ -3,8 +3,13 @@
   "version": "1.0.0",
```

Comment:
start with version 0.0.0 , if we start with version 1.0.0 the semantic release will break